va/va.go

```diff
@@ -620,6 +620,15 @@ func (va *ValidationAuthorityImpl) doRemoteOperation(ctx context.Context, op rem
 			firstProb = currProb
 		}
 
+		// If enough perspectives have passed, start a timer for how long we'll wait for remaining perspectives
+		if len(passed) >= required && len(passedRIRs) >= requiredRIRs {
+			// TODO: configurable
+			go func() {
```

Comment:
This should probably use time.AfterFunc, and the timer cancelled if we return without hitting this timeout